linkerd/tls/src/main/scala/io/buoyant/linkerd/tls/TlsUtils.scala

```diff
@@ -1,6 +1,8 @@
 package io.buoyant.linkerd.tls
 
-import java.io.File
+import java.io.{File, FileWriter}
+
```

Comment:
Linkerd import style is all imports grouped together (no blank lines) in alphabetical order.